state.go

```diff
@@ -92,78 +90,39 @@ func (s state) validate() (bool, string) {
 
 		if s.Settings.ClusterURI != "" && !s.Settings.BearerToken {
 			if !caCrt || !caKey {
-				return false, "ERROR: certificates validation failed -- You want me to connect to your cluster for you " +
+				return false, "certificates validation failed -- You want me to connect to your cluster for you " +
```

Comment:
woudl remove the "personnification" here too.

Reply:
:+1: